### DISCUSSION TITLE
Mi láser no Enciende

### DISCUSSION THREAD

**Discussion by wilson740** [arkypita/LaserGRBL#1109] [Q&A]
hola
Recién compre un cortador láser de 2500 mw  que no prende, tiene todo conectado, hace todos los movimientos de grabar pero No funciona el láser, no enciende, no entiendo la razón
Ayuda por favor que debo hacer
Gracias

**Comment by Morioka-san** [comment]
Hola, 
Lo tienes con M3 o M4? A mi pasó algo similar la primera vez que lo encendí, pero fue porque para M4, lo tenia en 225 de potencia en vez de 1000, y no quemaba nada. Cuando lo subí a 1000, funcionó bien.
